integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/DomainUtils.java

```diff
@@ -662,7 +662,7 @@ public static Domain createDomainResourceForDomainInImage(String domainUid,
         .spec(new DomainSpec()
             .domainUid(domainUid)
             .domainHome(WDT_IMAGE_DOMAINHOME_BASE_DIR + "/" + domainUid)
-            .dataHome("/u01/oracle/mydata")
+            .dataHome("/u01/mydata")
```

Comment:
This is dataHome override tests. The dataHome is set to /u01/oracle/mydata on purpose. The test will fail if it is set to /u01/mydata.